tests/virt_autotest/kubevirt_tests_server.pm

```diff
@@ -512,7 +533,11 @@ EOF
     my ($container_prefix, $container_tag, $pre_util_container_reg, $pre_util_container_tag);
     if ($kubevirt_ver ge "0.50.0") {
         $container_prefix = "$local_registry_fqdn:5000";
+        # Set the variable CONTAINER_TAG based on installed kubevirt version
+        set_var('CONTAINER_TAG', (split('-', $kubevirt_ver))[0]);
         $container_tag = get_required_var('CONTAINER_TAG');
+        # Check if the container tag exists in local private registry
+        assert_script_run("curl $container_prefix/v2/alpine-container-disk-demo/tags/list | jq -r '.tags[]' | grep $container_tag");
```

Comment:
Could you fix this problem -- https://openqa.suse.de/tests/18710394#step/kubevirt_tests_server/269, and then kick a new validation run that can check further code path?